components/lib/datatable/BodyCell.js

```diff
@@ -14,6 +14,67 @@ import { DomHandler, IconUtils, ObjectUtils, classNames } from '../utils/Utils';
 import { RowCheckbox } from './RowCheckbox';
 import { RowRadioButton } from './RowRadioButton';
 
+function getNestedValue(obj, path) {
```

Comment:
These 3 functions probably can be moved to ObjectUtils since they are generic functions.

Reply:
Done :). Also verified that there is no bug related to the metakey when sorting.